style/js/shield_defs.js

```diff
@@ -232,6 +232,30 @@ export function loadShields(shieldImages) {
     },
   };
 
+  shields["CA:QC:A"] = {
+    backgroundImage: [
+      shieldImages.shield40_ca_qc_a_2,
+      shieldImages.shield40_ca_qc_a_3,
+    ],
+    textColor: "white",
+    padding: {
+      left: 2,
+      right: 2,
+      top: 5.5,
+      bottom: 4,
+    },
+  };
+  shields["CA:QC:R"] = {
+    backgroundImage: shieldImages.shield40_ca_qc_r,
+    textColor: "white",
+    padding: {
+      left: 2,
+      right: 2,
+      top: 2,
```

Comment:
[Regional route shields on guide signs](https://commons.wikimedia.org/wiki/File:A20_Ouest_km143.jpg) keep the fleurs-de-lis, but I agree that omitting them looks cleaner at this size. Should we add a white horizontal line going through the top, nodding to the compartment that would normally contain the fleurs-de-lis? I’ve been thinking we’d want to do something similar for Maryland’s state route shields.

Reply:
Perhaps. I sent another message to talk-ca@ asking for feedback. Hopefully some locals will have an opinion on this.

Reply:
I suggest you give adding the white line a try and produce pictures to compare.

Reply:
Le voilà :

![Screenshot from 2022-04-11 09-46-09](https://user-images.githubusercontent.com/1732117/162753448-e3cef1fc-9588-40f5-8e31-0b2bf786544e.png)
![Screenshot from 2022-04-11 09-46-42](https://user-images.githubusercontent.com/1732117/162753450-01434eb8-6e8a-413b-b32c-75f9a922c4e6.png)

Reply:
For reference, the similar design elements on the Saskatchewan shields are removed: #271

Reply:
I like it personally. Thanks for doing this work.

Reply:
The additional white line and extra space looks good.  The real world shields are quite tall and the extra element helps suggest that.  Might make sense to do the same for the Saskatchewan shields too.  I've never been there, though, so I'm not sure.